spec/tbd_tests_spec.rb

```diff
@@ -1382,4 +1382,673 @@
       expect(surface[:heatloss]).to be_within(0.01).of(3.5)
     end
   end # KEEP
+
+  it "can check for attics vs plenums" do
```

Comment:
(Rather extensive) testing of attics vs plenums, at different stages of OSM development (w/w/o uprating).